pkg/sqlite/scene.go

```diff
@@ -1615,5 +1616,26 @@ func (qb *SceneStore) FindDuplicates(ctx context.Context, distance int) ([][]*mo
 		}
 	}
 
+	sortByPath(duplicates)
+
 	return duplicates, nil
 }
+
+func sortByPath(scenes [][]*models.Scene) {
+	lessFunc := func(i int, j int) bool {
+		firstPathI := getFirstPath(scenes[i])
+		firstPathJ := getFirstPath(scenes[j])
+		return firstPathI < firstPathJ
+	}
+	sort.SliceStable(scenes, lessFunc)
+}
+
+func getFirstPath(scenes []*models.Scene) string {
+	var firstPath string
+	for _, scene := range scenes {
+		if firstPath == "" || scene.Path < firstPath {
+			firstPath = scene.Path
+		}
+	}
+	return firstPath
```

Comment:
This is not exactly correct 
If some of the scenes contain an empty `""` path (file-less scenes ?) then the `firstPath` will be wrong
It might be ok for the current use case but if someone wants to re-use the function/s ( mostly sortByPath ) the prerequisites/exact use-case you mention will not be noticed.
For example with paths  "/home/a.mp4" "/home/b.mp4" "" "/home/c.mp4" "/home/d.mp4" you will  get "/home/c.mp4" as the first path

Changing the if condition in your code should be enough
```golang
	var firstPath string
	for i, scene := range scenes {
		if i == 0 || scene.Path < firstPath {
			firstPath = scene.Path
		}
	}
	return firstPath
```

Reply:
Fixed that.

Phew, I think this time it's finally correct :)